sormas-backend/src/main/resources/utils/deleteCases.sql

```diff
@@ -0,0 +1,156 @@
+DROP TABLE IF EXISTS delete_cleanup;
```

Comment:
Add a comment block at the start (draft, maybe adjust and add more info if you have it in mind):
-- Deletes all cases that are marked as deleted = TRUE.
-- All linked data will also be deleted, including contacts and event participants linked to the respective case. The person will be deleted if no other entity references it.